src/Candidate.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+
+function Candidate({ name, voteCount, cid, rank, vote }) {
```

Comment:
이렇게 바뀌어야겠죵?
`candidate`에서 또 비구조화할당해서 뽑아서 사용해주세요!

```suggestion
function Candidate({ candidate, rank, updateVoteCount }) {

  const {id,name,voteCount}=candidate;
```